gridftp/server/src/modules/file/globus_gridftp_server_file.c

```diff
@@ -1046,8 +1046,13 @@ globus_l_gfs_file_stat(
         stat_count = i;
         
         closedir(dir);
-        
+        goto done_fake;
 
+error_stat2:
+        globus_l_gfs_file_destroy_stat(stat_array, stat_count);
+error_alloc2:
+        closedir(dir);
+        goto error_open;
```

Comment:
...but that here I don't understand. Why can this be moved here?

Reply:
The labels error_stat2 and error_alloc2 are only used (in goto statements) inside the #ifdef WIN32 block. If the labels themselves are outside the #ifdef WIN32 block they become unused labels in the non-WIN32 case. This is the reason for moving the labels inside the #ifdef WIN32 block.

Reply:
Ah, ok, I didn't recognize that when comparing the old and current version of that file.